guides/common/modules/proc_creating-hosts-with-uefi-http-boot-provisioning.adoc

```diff
@@ -1,4 +1,4 @@
-[id="Creating_Hosts_with_UEFI_HTTP_Boot_Provisioning_{context}"]
+[id="Creating_Hosts_with_UEFI_HTTP_Boot_Provisioning"]
```

Comment:
If we already change it:

```suggestion
[id="creating-hosts-with-uefi-http-boot-provisioning"]
```

On "master", there's only on link in `guides/common/modules/con_prerequisites-for-using-ipxe.adoc`.